modules/3d-tiles/src/tz3/tz3-archive.ts

```diff
@@ -47,7 +50,7 @@ export class TZ3Archive {
     }
     const decompressedFile = Buffer.from(data);
 
-    return decompressedFile;
+    return decompressedFile.buffer;
```

Comment:
`this.getFileBytes` returns `ArrayBuffer`. If you want to handle `null`, it is a weird way. I'd recomment `throw` in null case. That means that the file has not been found